images/doppler-kubernetes-operator/tests/smoke_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+sleep 5
+
+DELAY=5
+NAMESPACE="doppler-operator-system"
+POD_SELECTOR="doppler-operator-controller-manager"
+TIMER=0
+TIMEOUT=60
+
+# Loop until the current time is greater than the end time
+while [ $TIMER -lt $TIMEOUT ]; do
+  # Run the command and check for the pod is running
+  STATUS=$(kubectl get pods -n "$NAMESPACE" -o jsonpath='{.items[*].status.phase}')
```

Comment:
i think we don't need this smoke script since Helm by default asserts this case with `wait: true` arg. See: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#argument-reference

Reply:
I changed the smoke test to create a generic token